test/io-tests/fixtures.yaml

```diff
@@ -1,3 +1,144 @@
+cli:
+# success: valid commands
+  - name: help long
+    args: ['--help']
+  - name: help short
+    args: ['-h']
+  - name: example long
+    args: ['--example']
+  - name: example short
+    args: ['-e']
+  - name: dump config
+    args: ['--dump-config']
+    use_defaultenv: true
+  - name: dump schema
+    args: ['--dump-schema']
+    use_defaultenv: true
+# failures: config files
+  - name: no config
+    expect: error
+  - name: non-existant config file
+    expect: error
+    args: ['does_not_exist.conf']
+  - name: invalid config file
+    expect: error
+    args: ['test/io-tests/configs/invalid.yaml']
+# failures: required config options
+  - name: missing db-anon-role
+    expect: error
+    env:
+      PGRST_DB_URI: required
+      PGRST_DB_SCHEMAS: required
+  - name: missing db-schemas
+    expect: error
+    env:
+      PGRST_DB_ANON_ROLE: required
+      PGRST_DB_URI: required
+  - name: missing db-uri
+    expect: error
+    env:
+      PGRST_DB_ANON_ROLE: required
+      PGRST_DB_SCHEMAS: required
+# failures: wrong config values
+  - name: invalid server-unix-socket-mode not octal
+    expect: error
+    use_defaultenv: true
+    env:
+      PGRST_SERVER_UNIX_SOCKET_MODE: '800'
+  - name: invalid server-unix-socket-mode < 600
+    expect: error
+    use_defaultenv: true
+    env:
+      PGRST_SERVER_UNIX_SOCKET_MODE: '599'
+  - name: invalid server-unix-socket-mode > 777
+    expect: error
+    use_defaultenv: true
+    env:
+      PGRST_SERVER_UNIX_SOCKET_MODE: '778'
+# TODO: Bug needs to be fixed
+#  - name: invalid jwt-aud
+#    expect: error
+#    use_defaultenv: true
+#    env:
+#      PGRST_JWT_AUD: 'htp:/@@localhorst.invalid'
```

Comment:
@wolfgangwalther Any string value is allowed in `jwt-aud`, I think this is not a bug.